WordPressVIPMinimum/Sniffs/UserExperience/AdminBarRemovalSniff.php

```diff
@@ -201,10 +201,8 @@ public function process_parameters( $stackPtr, $group_name, $matched_content, $p
 		switch ( $matched_content ) {
 			case 'show_admin_bar':
 				$error = true;
-				if ( true === $this->remove_only ) {
-					if ( 'true' === $parameters[1]['raw'] ) {
-						$error = false;
-					}
+				if ( ( true === $this->remove_only ) && 'true' === $parameters[1]['raw'] ) {
```

Comment:
Should we remove the extra set of brackets enclosing `true === $this->remove_only`?